Composer/packages/client/src/components/CreationFlow/v2/CreateBot.tsx

```diff
@@ -199,17 +199,22 @@ export function CreateBotV2(props: CreateBotProps) {
 
   useEffect(() => {
     const itemKey = selectedProgLang.props.itemKey;
+    let newTemplates: BotTemplate[] = [];
     if (itemKey === csharpFeedKey) {
-      const newTemplates = templates.filter((template) => {
+      newTemplates = templates.filter((template) => {
         return template.dotnetSupport;
       });
-      setDisplayedTemplates(newTemplates);
     } else if (itemKey === nodeFeedKey) {
-      const newTemplates = templates.filter((template) => {
+      newTemplates = templates.filter((template) => {
         return template.nodeSupport;
       });
-      setDisplayedTemplates(newTemplates);
     }
+    if (creationFlowType === 'Skill') {
+      newTemplates = newTemplates.filter((template) => {
+        return !template.isMultiBotTemplate;
```

Comment:
why don't you just check template.keywords.includes here?

Reply:
As it is now keywords are not passed to the front end template object, they are parsed on the server side and the server sets values like template.isMultiBotTemplate or template.dotnetSupport

Reply:
Like template.keywords doesnt exist